websites/L/Leboncoin/presence.ts

```diff
@@ -0,0 +1,110 @@
+const presence = new Presence({
+		clientId: "966800501326876692"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			smallImageKey: "reading",
+			smallImageText: "Regarde des annonces",
+			startTimestamp: browsingTimestamp
+		},
+		defaultPresenceData: PresenceData = {
+			largeImageKey: "logo",
+			smallImageKey: "reading",
+			smallImageText: "Regarde des annonces",
+			startTimestamp: browsingTimestamp,
+			details: "Regarde la page d'accueil"
+		};
+
+	if (document.location.hostname === "www.leboncoin.fr") {
+		if (document.location.pathname.includes("/deposer-une-annonce")) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Déposer une annonce";
+		} else if (document.location.pathname.includes("/mes-recherches")) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Mes recherches sauvegardées";
+		} else if (document.location.pathname.includes("/mes-annonces")) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Mes annonces sauvegardées";
+		} else if (document.location.pathname.includes("/emploi")) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Offres d'emploi";
+		} else if (
+			document.location.pathname.includes("/paiement-securise-livraison")
+		) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Paiement sécurisé";
+		} else if (document.location.pathname.includes("/messages")) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Messages privés";
+		} else if (document.location.pathname.includes("/dc")) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Informations légales";
+		} else if (document.location.pathname.includes("/annonces/offres")) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Annonces pour toute la France";
+		} else if (document.location.pathname.includes("/compte")) {
+			presenceData.details = "Regarde la page :";
+			presenceData.state = "Paramètres du compte";
+		} else if (document.location.pathname.includes("/profil/")) {
+			presenceData.details = "Regarde la page :";
+			const profilename = document.querySelector(
+				"#mainContent > div.styles_Profile__3vhTo > div.styles_ProfileHero__aQISv > div > div.styles_userInfo__XBcBm > div > h3"
+			).textContent;
+			presenceData.state = `Profil de ${profilename}`;
+			presenceData.buttons = [
+				{ label: "Consulter le profil", url: `${document.URL}` }
+			];
+		} else if (document.location.pathname.includes("/blog")) {
+			presenceData.details = "Viewing Page:";
+			presenceData.state = "Node.js News";
+		} else if (document.location.pathname.includes("/recherche")) {
+			presenceData.details = "Dans les résultats de recherche :";
+
+			presenceData.state = `Annonces pour « ${
```

Comment:
do you really wanna use «? What's the goal with that character?

Reply:
« » (guillemets) is the french version of " ". I thought it was better to use it since the rest of the sentence is in french.